### DISCUSSION TITLE
GitHub Classroom UI is buggy and 404 error when syncing with Google Classroom

### DISCUSSION THREAD

**Discussion by IonutCicio** [community/community#120545] [GitHub Education]
### Hi everyone,

Yesterday I wanted to try for the first time GitHub Clasroom as it would be an an amazing tool in my case, but I met some problems.

For example, everytime I open a dialog (i.e. Sync students from Google Classrom, edit tests, search repository etc...) the background of the dialog is transparent.

![image](https://github.com/community/community/assets/40034429/e7796513-b7aa-4bd8-bc34-8708a5a54d8a)

The other problem is that when I try to sync the roster, I'm greeted by a 404 error

![image](https://github.com/community/community/assets/40034429/9bfed207-0b92-4e31-963e-1a5b552b7960)

The troublesome link is [https://classroom.github.com/classrooms/classroom-name/roster/sync_google_classroom](https://classroom.github.com/classrooms/classroom-name/roster/sync_google_classroom)

Hope it gets fixed soon, can't wait to start using GitHub Classroom!

**Comment by chrissphinx** [comment]
If you want a workaround, using a plugin to apply user stylesheets to specific URLs, you can target this:
```
anchored-position:not(.Overlay) {
    background-color: red;
}
```
for whatever dumb reason the background-color on these elements are being computed to be `rgba(0, 0, 0, 0)`

**Reply by IonutCicio** [reply]
They fixed this bug. The 404 when syncing a roster still remains.
